packages/itwin/tree-widget/src/e2e-tests/utils.ts

```diff
@@ -165,7 +165,9 @@ export async function expandNode(page: Page, node: Locator) {
   await page.keyboard.press("ArrowRight");
 }
 
-export async function openErrorDropdown(treeWidget: Locator, errorAmount: number = 1) {
+export async function openErrorDropdown(page: Page, treeWidget: Locator, errorAmount: number = 1) {
   const dropdown = treeWidget.getByRole("button", { name: `${errorAmount} issues found` });
+  await dropdown.waitFor();
+  await takeScreenshot(page, treeWidget, { resetScroll: true });
```

Comment:
I don't think an "openErrorDropDown" function should be taking screenshots. Also, it seems when we click to expand it, we should `await` on some text to appear, no?

Reply:
based on `click` documentation `Wait for initiated navigations to either succeed or fail, unless [noWaitAfter] option is set.` I would believe there is no need to check for text, as the text does not have a loading state.

For the screenshot we could have an option `TakeIntermediateScreenshot`: boolean or rename the function. But I would like to keep the ability to screenshot the closed dropdown as we would like to screenshot all the errors in the future.

Reply:
Can we have something like this?
```ts
test("test", async () => {
  // ....
  const errorDropdown = await getErrorDropdown(page, treeWidget, 2);
  takeScreenshot();
  await errorDropdown.click();
  takeScreenshot();
});
```

Reply:
Yeah that works too, changed.